modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/JsonProcessor.java

```diff
@@ -56,11 +59,12 @@ boolean isAddToRoot() {
         return addToRoot;
     }
 
-    public static Object apply(Object fieldValue) {
+    public static Object apply(Object fieldValue, boolean allowDuplicateKeys) {
         BytesReference bytesRef = fieldValue == null ? new BytesArray("null") : new BytesArray(fieldValue.toString());
         try (InputStream stream = bytesRef.streamInput();
-             XContentParser parser = JsonXContent.jsonXContent
+             JsonXContentParser parser = (JsonXContentParser) JsonXContent.jsonXContent
```

Comment:
Indentation

Reply:
This is inside the try, so I believe the indentation is correct.

Reply:
Ah, you're right. I missed that.